vscode-client/README.md

```diff
@@ -20,7 +20,13 @@ npm i -g bash-language-server
 - [x] Highlight occurrences
 - [x] Code completion
 - [x] Simple diagnostics reporting
+- [x] Documentation for flags on hover
 - [ ] Rename symbol
 
+## Configuration
+
```

Comment:
I guess we need some more explanation here?

Reply:
Done

Reply:
Can you add a link to how to set up explainshell in Docker?

Reply:
My internet is back up and I pushed the pre-built image https://hub.docker.com/r/chrismwendt/codeintel-bash-with-explainshell/ And I added this to the docs

Reply:
Awesome. Maybe also a link to https://github.com/idank/explainshell/pull/125 ?

Reply:
Done